test/r2.md

```diff
@@ -439,3 +441,27 @@ The following command with broken `single_cortx.iso` should be failed
 ```bash
 provisioner set_swupgrade_repo 2.0.1 --source=/opt/iso/single_cortx.iso
 ```
+
+
+## EOS-17828: Port adjustments post discussions for 3 node
+
+Step 1: Ensure firewall component is present in all the nodes of a 3 node VM setup. If not, install with the below command
+```bash
+salt '*' state.apply components.system.firewall
+```
+
+Step 2: Verify the services with corresponding ports, are mapped to all the zones with the below command.
+```bash
+firewall-cmd --list-all
+```
+
+Step 3: Verify if the services are mapped to the zones on all the nodes.
+```bash
+ssh srvnode-2 firewall-cmd --list-all
+ssh srvnode-3 firewall-cmd --list-all
+```
```

Comment:
Ideally, `EOS-18494: Validation for Open Ports in Firewall - Salt Execution Module` should be the only required deliverable for validation and the individual commands shall not be required.

Reply:
Yeah, but I couldn't find scenarios for EOS-18494 yet in `r2.md` . So, for now, added this as simple verification step.